src/windows/options.c

```diff
@@ -221,7 +221,7 @@ void window_options_open()
 	if (w != NULL)
 		return;
 
-	w = window_create_centred(WW, WH, (uint32*)window_options_events, WC_OPTIONS, 0);
+	w = window_create_centred(WW, WH, (uint32*)window_options_events, WC_OPTIONS, WF_STICK_TO_FRONT);
```

Comment:
This has the side effect that closing all windows via a shortcut (Shift + Backspace) doesn't close the Options window.